e2e/node/util/supportsAcp.ts

```diff
@@ -0,0 +1,24 @@
+/**
+* Copyright 2022 Inrupt Inc.
+*
+* Permission is hereby granted, free of charge, to any person obtaining a copy
+* of this software and associated documentation files (the "Software"), to deal in
+* the Software without restriction, including without limitation the rights to use,
+* copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+* Software, and to permit persons to whom the Software is furnished to do so,
+* subject to the following conditions:
+*
+* The above copyright notice and this permission notice shall be included in
+* all copies or substantial portions of the Software.
+*
+* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+* INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+* PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+* HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+* OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+
+export function supportsAcp(url: string) {
+  return url.includes("pod.inrupt.com");
```

Comment:
Wouldn't this be better explicitly defined as part of the environment ? E.g. adding `E2E_TEST_ACP` and `E2E_TEST_WAC` to the environments, instead of relying on IRI matching ?

Reply:
Fixed in https://github.com/inrupt/solid-client-js/pull/1426/commits/3f12c1a4758d580af685f89dd2c47b56ed52e841.